Add figure issue template (#11)
arfc/.github: .github/ISSUE_TEMPLATE/figure.md

---
name: Figure 
about: Create a figure for use in LaTeX
title: '[Figure] '
labels: 'Status:1-New, Type:Feature'
assignees: ''

---
# Description

> <!-- Describe what you want made. What is the image intended to show? -->

Reviewer: ```suggestion
<!-- Describe the figure. What is the image intended to show? -->
```

# Method
<!-- How will the figure be produced? ex. TikZ, Python -->
**Program:** 

<!-- Include any additional notes that may be useful for a developer. -->

# How can this issue be closed?
Issue can be closed when the following criteria has been met:

> - [ ] Figure is completely modular and all files necessary for generation and inclusion are confined to a single file or directory.

Reviewer 2: ```suggestion
- [ ] Figure is completely modular and reproducible.
```

Author: I think keeping the extra instructions are worthwhile, especially for new students.

- [ ] Includes a caption that is standalone and ends with correct punctuation.

> - [ ] Includes a legend.

Reviewer 2: ```suggestion
- [ ] Includes a legend (if needed).
```
Not all figures need a legend, depending on what it is showing (for example plotting keff over time may not need a legend if only one set of data is being plotted)

- [ ] All text in the figure is large enough to be easily read.
- [ ] Has a label (use `\label{}`)

> - [ ] Are your color choices appropriate for adequate accessibility? 

Reviewer: ```suggestion
- [ ] Are your color choices appropriate for [adequate accessibility](https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html#the-color-scales)? 
```
